.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+name: Automated Type and Format Checking
+
+on: [push]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
```

Should we use the current LTS?

```suggestion
          node-version: "14.x"
```